web/src/main/java/com/bazaarvoice/emodb/web/auth/DefaultRoles.java

```diff
@@ -126,11 +135,9 @@
 
     // Reserved role for anonymous access
     anonymous (
-            // TODO:  Lock this down.  For now this will permit all standard client operations.
-            //        Additionally, Shovel historically did not require permission to create facades, only to
-            //        update facade records.  To maintain backwards compatibility anonymous also has this permission.
-            ImmutableSet.of(standard),
-            Permissions.createFacade(ALL));
+            // TODO:  Lock this down.  For now this will permit all standard client operations with excluding pii tables/placements in SOR.
+            // note that we are excluding the permissions to the pii tables/placements in SOR.
```

Comment:
Nitpick:  This line is redundant, it's nearly identical to the previous TODO line.